File: app/src/main/java/com/simpletodo/aayushf/simpletodo/DrawerListViewPopulator.java
package com.simpletodo.aayushf.simpletodo;

import android.content.Context;
import android.support.annotation.LayoutRes;
import android.support.annotation.NonNull;
import android.widget.ArrayAdapter;

/**
 * Created by ayfadia on 5/18/17.
 */

public class DrawerListViewPopulator extends ArrayAdapter<TagItem> {
    public DrawerListViewPopulator(@NonNull Context context, @LayoutRes int resource) {
        super(context, resource);
    }
}

File: app/src/main/java/com/simpletodo/aayushf/simpletodo/DrawerListViewPopulator.kt
package com.simpletodo.aayushf.simpletodo

import android.content.Context
import android.support.annotation.LayoutRes
import android.widget.ArrayAdapter

/**
 * Created by ayfadia on 5/18/17.
 */

class DrawerListViewPopulator(context: Context, @LayoutRes resource: Int) : ArrayAdapter<TagItem>(context, resource)
